app/page.tsx

```diff
@@ -6,43 +6,64 @@ import {
   Grid,
   GridItem,
   Flex,
+  Text,
   useColorModeValue
 } from '@chakra-ui/react'
+import { useRouter } from 'next/navigation'
 import { PrimaryButton } from '@/components/button'
+import { Loading } from '@/components/loading'
 import { Header, Footer } from '@/components/navigation'
 import { TripSearch, TripSort, TripCard } from '@/trip/components'
+import { useTripsCollectionQuery, TripsOrderBy } from '@generated/api'
 
-const dummy = [
-  {
-    id: 1,
-    title: 'Tokyo',
-    date_from: 'Oct 10, 2023',
-    date_to: 'Nov 10, 2023',
-    image_storage_object_id:
-      'https://placehold.jp/6381f8/ffffff/366x277.png?text=Picture',
-    share: [
-      { id: '', image: '' },
-      { id: '', image: '' }
-    ]
-  },
-  {
-    id: 2,
-    title: 'Kyoto',
-    date_from: 'Oct 10, 2023',
-    date_to: 'Nov 10, 2023',
-    image_storage_object_id:
-      'https://placehold.jp/6381f8/ffffff/366x277.png?text=Picture',
-    share: [
-      { id: '', image: '' },
-      { id: '', image: '' }
-    ]
-  }
-]
-
-export default function Top() {
+export default function Top({ searchParams }: { searchParams: { q: string } }) {
+  const router = useRouter()
   const bg = useColorModeValue('white', 'gray.800')
   const color = useColorModeValue('black', 'gray.300')
 
+  const searchWord = searchParams.q
+
+  const { data, loading, fetchMore, refetch } = useTripsCollectionQuery({
+    variables: {
+      filter: {
+        user_id: { eq: 1 },
```

Comment:
[question] How is the user_id fetched? If user_id is static ID in this PR, I suggest you add comment like below.

```typescript
filter: {
  // TODO: Fix dynamic user_id
  user_id: { eq: 1 }
```

Reply:
Thank you! You're right and I added the comment 👍 
https://github.com/tabi-memo/frontend/pull/11/commits/d45fc1868e31d17157ef313d16771403beb40513